scaler/scaler.go

```diff
@@ -9,10 +9,19 @@ import (
 )
 
 type ScaleInParams struct {
```

Comment:
Could these be merged in to `ScaleParams`? They seem symmetric with each other.

Reply:
Yeah, probably. I kind of liked having the type safety and in the current world of positive vs negative changes there didn't seem much economy to combining them.